riscv-debug-spec.tex

```diff
@@ -792,6 +796,11 @@ \subsection{JTAG Connector}
 
 \subsection{JTAG Registers}
 
+%MAW -- is it required that the DTM be the only functionality of the TAP?
```

Comment:
I assumed that the same TAP can also be used for basic scan chain functionality, which is why I left all that IR space for "standard" JTAG instructions. The 5-bit IR is a minimum.